### PR TITLE
Changed tag selection widget to checkboxes in institution edit page.

### PR DESCRIPTION
**Pull request by dzejkobi**

### CODE REVIEW

**Comment by ad-m** [comment]
@kaxil, what is missing here to label `backend`? According to 
https://github.com/watchdogpolska/feder/blob/a931b4fa2c72caa5002999f3284b16e58fde61a2/.github/boring-cyborg.yml#L12-L13 I believe it should be labeled as `backend`, because it touch file `feder/institutions/forms.py`.

File: feder/institutions/forms.py [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+class InstitutionForm(UserKwargModelFormMixin, forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Form "helper" is needed to have control over rendering by "crispy" tag.
+        self.helper = FormHelper()
+        self.helper.layout = Layout(
+            Fieldset("", "name", "jst", "regon", "email"),
+            Fieldset("", "tags", css_class="form-group scrollable-widget"),
+            ButtonHolder(Submit("submit", _("Update"), css_class="btn btn-primary")),
+        )
```
Comment (ad-m) [inline feder/institutions/forms.py:20]: I suggest follow notation from other part of project: https://github.com/watchdogpolska/feder/blob/a25820deea70d09187e6fc17e91046826d0de475/feder/monitorings/forms.py#L19-L22
It allows skip `FormHelper` implicit definition and use implicit submit button definition.

Comment (dzejkobi) [reply feder/institutions/forms.py:16]: Ok, change applied.

**Comment by kaxil** [comment]
> @kaxil, what is missing here to label `backend`? According to
> https://github.com/watchdogpolska/feder/blob/a931b4fa2c72caa5002999f3284b16e58fde61a2/.github/boring-cyborg.yml#L12-L13
> 
> I believe it should be labeled as `backend`, because it touch file `feder/institutions/forms.py`.

@ad-m `**/.py` if you want to match all python files in the project. `/*.py` (or just `*.py`) if you want to match all python files in root directory.

**Review by ad-m — APPROVED** [review]
Smells good. I did not run it, we will test it in production.

**Comment by ad-m** [comment]
@kaxil, thank for you help. It works now!

**Pull request merged**